src/kebab.rs

```diff
@@ -17,9 +18,19 @@ pub trait ToKebabCase: ToOwned {
     fn to_kebab_case(&self) -> Self::Owned;
 }
 
+pub fn to_kebab(s: &str, number_starts_word: bool) -> String {
+    transform(s, number_starts_word, lowercase, |s| s.push('-'))
+}
+
 impl ToKebabCase for str {
     fn to_kebab_case(&self) -> Self::Owned {
-        transform(self, lowercase, |s| s.push('-'))
+        convert_case(
+            &self,
+            ConvertCaseOpt {
+                case: Case::Kebab,
+                number_starts_word: false,
+            },
+        )
```

Comment:
With the proposed changes to the `convert_case` module, this implementation simply becomes

```suggestion
        self.convert_case(Case::Kebab)
```

Reply:
Same for the other modules.